src/components/filesystem/index.js

```diff
@@ -22,36 +22,40 @@ module.exports = {
             modalLinks:[],
             showShare:false,
             showSharedWith:false,
-            sharedWithData:{"edit_shared_with_users":[],"read_shared_with_users":[]},
-	    forceSharedWithUpdate:0,
-	    isNotBackground: true,
+            sharedWithData:{},
```

Comment:
Looks like a bad merge on this line. Lost "edit_shared_with_users":[],"read_shared_with_users":[]

Reply:
done